.github/labeler.yml

```
@@ -1,15 +1,25 @@
+---
 # Add 'documentation' label to any change to *.md files
 documentation:
-- any: ['**/*.md']
+    - changed-files:
+          - any-glob-to-any-file: '**/*.md'
 
 # Add 'docker' label to any change in docker related files
 docker:
-- any: ['**/Dockerfile*', '**/.dockerignore']
+    - changed-files:
+          - any-glob-to-any-file: ['**/Dockerfile*', '**/.dockerignore']
 
 # Add 'github_actions' label to any change .github/ directory
 github_actions:
-- any: ['.github/**']
+    - changed-files:
+          - any-glob-to-any-file: .github/**
```

This one looks broken. It should be in a list like the others, right?

```suggestion
          - any-glob-to-any-file: [.github/**]
```

---

So I tested it, and it did properly add the label and it's also valid [per the documentation](https://github.com/marketplace/actions/labeler#basic-examples) Also, the yaml formatter pre-commit auto removes them.